eras/alonzo/test-suite/src/Test/Cardano/Ledger/Alonzo/Translation/TranslationInstance.hs

```diff
@@ -73,6 +73,9 @@ instance Cborg.Serialise PV1.TxInInfo
 instance Cborg.Serialise PV1.TxInfo
 instance Cborg.Serialise PV1.TxOut
 instance Cborg.Serialise PV2.ScriptPurpose
+instance Cborg.Serialise PV2.TxId
+instance Cborg.Serialise PV2.TxOutRef
+instance Cborg.Serialise PV2.TxInInfo
```

Comment:
These types existed before in V2, and V3 was using them, but not anymore. Now V3 has its own copies of these types, with the only difference being that copies use V3.TxId instead of the V2.TxId